openmetadata-ui/src/main/resources/ui/src/components/common/AsyncSelectList/AsyncSelectList.tsx

```diff
@@ -174,7 +174,7 @@ const AsyncSelectList: FC<AsyncSelectListProps & SelectProps> = ({
   };
 
   const dropdownRender = (menu: React.ReactElement) => (
-    <>
+    <div data-testid="tags-dropdown">
```

Comment:
Remove div as it's seems to be test id purpose, Do not make DOM heavy for the tests. Find alternative for assertion